### PR TITLE
chore: migrate to the Tool Kit beta

### PR DESCRIPTION
**Pull request by rowanmanning**
The beta is way faster and Platforms need more people to test it. Everything seems to work and I think this would be possible to automate for consumers that don't specify custom plugins (note: both of the plugins in this repo are candidates for inclusion in Tool Kit itself).

### CODE REVIEW

**Review by ivomurrell — APPROVED** [review]
🎉 thank you for collaborating with us on this! we'll be happy to field any bug reports or ux issues so we can get the release as polished as possible

File: .toolkit/plugins/commitlint/.toolkitrc.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
-hooks:
-  git:commitmsg:
-    - Commitlint
+tasks:
+  Commitlint: './index.js'
```
Comment (joelcarr) [inline .toolkit/plugins/commitlint/.toolkitrc.yml:2]: **question:** @ivomurrell is it possible to extend this plugin to run on CI?

Comment (rowanmanning) [reply .toolkit/plugins/commitlint/.toolkitrc.yml:2]: It's possible yeah, [here's the command](https://github.com/rowanmanning/actions/blob/95bd5a6df32e7304563a46513e0efa026ea91366/.github/workflows/enforce-commit-style.yml#L21-L24)

Comment (ivomurrell) [reply .toolkit/plugins/commitlint/.toolkitrc.yml:2]: yes! if they wanted it to run in CI they could set the task to run in the `test:ci`

**Review by CyntiBinti — APPROVED** [review]

**Pull request merged**